Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -64,9 +67,11 @@ struct FPTIBatchData: Codable {
             endTime: Int? = nil,
             errorDescription: String? = nil,
             eventName: String,
+            merchantExperiment: String? = nil,
```

Comment:
nit to just move this above `paymentMethodsDisplayed` to be alphabetical, will also want to update the rest of this file with that order.